CRM/Core/DAO.php

```diff
@@ -2769,6 +2778,8 @@ public static function acceptedSQLOperators() {
       'NOT IN',
       'BETWEEN',
       'NOT BETWEEN',
+      'IS',
+      'IS NOT',
```

Comment:
I feel reluctant about adding this to CRM_Core_DAO.
This will affect APIv3 not just v4, and probably some other things as well like search builder.
Probably better to create our own list of accepted operators for APIv4.